server/sonar-web/src/main/js/components/controls/SearchBox.tsx

```diff
@@ -145,7 +147,9 @@ export default class SearchBox extends React.PureComponent<Props, State> {
           value={value}
         />
 
-        <SearchIcon className="search-box-magnifier" />
+        <DeferredSpinner loading={loading !== undefined ? loading : false}>
```

Comment:
the "x" icon is no more aligned on (at least) Chrome:

![image](https://user-images.githubusercontent.com/2317341/36585565-b050b06e-187e-11e8-9ac9-3e6f7ca33c8d.png)